roles/ood_user_reg_cloud/templates/user-reg_conf.j2

```diff
@@ -1,13 +1,24 @@
-LoadModule proxy_uwsgi_module modules/mod_proxy_uwsgi.so
-
 ProxyPass /socket.io/ http://127.0.0.1:8000/socket.io/
 ProxyPassReverse /socket.io/ http://127.0.0.1:8000/socket.io/
 
-ProxyPass /register http://127.0.0.1:8000/register/
+ProxyPass /register http://127.0.0.1:8000/
+ProxyPassReverse /register http://127.0.0.1:8000/
+
```

Comment:
Could we host the register app in the proxy space at "http://127.0.0.1:8000/register/"?  It seems like we could be consistent with the prior uwsgi config an not have to change the flask app to the root of the container.